tests/Aspire.Npgsql.EntityFrameworkCore.PostgreSQL.Tests/TestDbContext.cs

```diff
@@ -5,7 +5,12 @@
 
 namespace Aspire.Components.Common.Tests;
 
-public class TestDbContext : DbContext
+public interface ITestDbContext
```

Comment:
To test we can register and Enrich a DbContext with distinct service and implementation.